myosuite/envs/myo/myochallenge/bimanual_v0.py

```diff
@@ -246,6 +246,7 @@ def get_reward_dict(self, obs_dict):
         obj_pos = obs_dict["obj_pos"][0][0] if obs_dict['obj_pos'].ndim == 3 else obs_dict['obj_pos']
         palm_pos = obs_dict["palm_pos"][0][0] if obs_dict["palm_pos"].ndim == 3 else obs_dict["palm_pos"]
         goal_pos = obs_dict["goal_pos"][0][0] if obs_dict["goal_pos"].ndim == 3 else obs_dict["goal_pos"]
+        goal_pos = np.concatenate((goal_pos[:2], np.array([1.09])))
```

Comment:
Could you explain why are we concatenating a fixed value? If it doesn't change then doesn't it just lead to wasted computations for RL?

Reply:
Nevermind, I see that it is only for the local variable

Reply:
what is the "1.09" value about? can this added as a constant variable

Reply:
This is the constant pillar height used for evaluation. Committed here as a constant variable: [ae09250](https://github.com/MyoHub/myosuite/pull/258/commits/ae09250d167708a6a6d3b153b3c00310efe6909d)